lib/sonatype-nexus3-stack.ts

```diff
@@ -387,7 +387,7 @@ export class SonatypeNexus3Stack extends cdk.Stack {
           },
         },
         nexusProxy: {
-          enabled: true,
+          enabled: false,
```

Comment:
since disabling `nexusProxy`, should we also remove other properties for nexus proxy?

Reply:
We still need to use other properties, for example, nexusHttpHost to create ingress rule if domainName is provided.  Here is the reference [ingress template](https://github.com/Oteemo/charts/blob/master/charts/sonatype-nexus/templates/ingress.yaml) for more details.